### DISCUSSION TITLE
Explorer

### DISCUSSION THREAD

**Discussion by FromCppToCarbon** [carbon-language/carbon-lang#2297] [Getting started]
How do we follow updation for testdata ? 
Do we have to re-clone  carbon language with git once every sometime ? 
Testdata are  being updated online but not locally .

**Comment by chandlerc** [comment]
The `testdata` directories are normal directories in the git repo -- any `git pull` or other update to the repo should update `testdata` if there are relevant updates.

That said, some of these are only updated periodically to rebuild the seed corpus for the fuzzers. That's normal -- they don't need to be updated in most cases as they just serve as a baseline seed for use when doing continuous fuzzing. Some folks run fuzzers continuously, seeded with these inputs, in order to find bugs (including new bugs).

**Comment by FromCppToCarbon** [comment]
Thanks a lot Chandlerc .
I will try to follow up and learn from the available updated docs .
